middleware/session/store.go

```diff
@@ -72,6 +72,33 @@ func (s *Store) Get(c *fiber.Ctx) (*Session, error) {
 	return sess, nil
 }
 
+// getSessionID will returns the session id from:
+// 1. cookie
+// 2. http headers
+// 3. query string
+func (s *Store) getSessionID(c *fiber.Ctx) string {
+	id := c.Cookies(s.CookieName)
+	if len(id) > 0 {
+		return id
+	}
+
+	if s.Source == SourceHeader {
+		id = string(c.Request().Header.Peek(s.CookieName))
```

Comment:
We shouldn't use the cookie name as the header name, this will be confusing to users. Instead we should use a key-value syntax, see our csrf middleware for an example: https://github.com/gofiber/fiber/blob/master/middleware/csrf/csrf.go#L30-L40

Reply:
I have been added `KeyLookUp` to instead it.